.github/workflows/codeql-query.yml

```diff
@@ -66,13 +66,17 @@ jobs:
         shell: node {0}
         run: |
           const fs = require('fs');
+          const process = require('process');
           const allRepos = JSON.parse(fs.readFileSync("instructions.json")).repositories;
           const repoNwos = new Set(${{ toJSON(matrix.repoNwos) }});
           const repositories = allRepos.filter(r => repoNwos.has(r.nwo));
           if ("${{ secrets.TEST_PAT }}") {
             allRepos.forEach(r => r.pat = "${{ secrets.TEST_PAT }}")
           }
-          console.log(`::set-output name=repositories::${JSON.stringify(repositories)}`);
+          const githubOutput = process.env.GITHUB_OUTPUT;
+          const outputFile = fs.openSync(githubOutput, 'a');
+          fs.writeSync(outputFile, `repositories=${JSON.stringify(repositories)}`);
+          fs.closeSync(outputFile);
```

Comment:
`@actions/core` seems to be using a single method for issuing these calls: https://github.com/actions/toolkit/blob/b36e70495fbee083eb20f600eafa9091d832577d/packages/core/src/file-command.ts#L22-L24. Can we also use `fs.appendFileSync` here instead of making these three separate calls (if the `require('@actions/core')` above doesn't work)?